Source/WebCore/rendering/RenderLayerScrollableArea.cpp

```diff
@@ -1941,4 +1941,12 @@ void RenderLayerScrollableArea::didStartScrollAnimation()
     m_layer.page().scheduleRenderingUpdate({ RenderingUpdateStep::Scroll });
 }
 
+void RenderLayerScrollableArea::animatedScrollDidEnd()
+{
+    if (m_needsAnimatedScroll) {
```

Comment:
Don't we need similar changes for animated scrolls in `<iframe>`s?

Reply:
Per our discussion and the test I don't think so.